pkg/cloud/volume_limits.go

```diff
@@ -551,3 +556,20 @@ var gpuInstanceGpus = map[string]int{
 	"p4de.24xlarge": 8,
 	"p5.48xlarge":   8,
 }
+
+// / Note this table is not a reflection of how many accelerators an instance has but of how many slots their combined acclerators take up
```

Comment:
@ConnorJC3 @torredil Is there a reason we are doing `// /` for comments in this file? 

If not, can we delete all the extra `/`'s?

Reply:
No special reason I'm aware of, if we get rid of them preferably do the whole file for consistency.

Reply:
Agreed on delete for whole file